lib/fog/libvirt/models/compute/server.rb

```diff
@@ -346,13 +346,13 @@ def to_xml
 
                 volumes.each_with_index do |volume, index|
                   target_device = "vd#{('a'..'z').to_a[index]}"
-                  if ceph_args && volume.pool_name.include?(ceph_args["libvirt_ceph_pool"])
+                  if ceph_args && ceph_args["libvirt_ceph_pool"]&.split(",")&.include?(volume.pool_name)
```

Comment:
I'd prefer to fix this up in `read_ceph_args` so you don't do this in a loop. Same below for the monitor. Perhaps also add in ` .map(&:strip)` so you can also use spaces between the values while you're at it.

Another thing is that `libvirt_ceph_pools` suggests it's singular while so perhaps store the result of that as `libvirt_ceph_pools` to indicate it's an array?

Reply:
Ok ! I have updated the branch to move the split and verify keys in the read_ceph_args.